checkov/common/bridgecrew/platform_integration.py

```diff
@@ -554,6 +586,8 @@ def persist_scan_results(self, scan_reports: list[Report]) -> None:
         dpath.merge(reduced_scan_reports, checks_metadata_paths)
         persist_checks_results(reduced_scan_reports, self.s3_client, self.bucket, self.repo_path)
```

Comment:
Do the sast reports need to be removed from `reduced_scan_reports`?

Reply:
Yes. We persist it here - `persist_sast_checks_results`